src/encoded/static/libs/ui/tooltip.js

```diff
@@ -67,16 +68,32 @@ const Tooltip = (props) => {
     const positionTooltip = React.useCallback(() => {
         // Determining the position of the tooltip trigger and accounting for vertical scroll
         const scrollTop = (window.pageYOffset !== undefined) ? window.pageYOffset : (document.documentElement || document.body.parentNode || document.body).scrollTop;
-        const viewportWidth = window.innerWidth || document.documentElement.clientWidth;
+        let viewportWidth = 0;
+        if (document.getElementById('content')) {
```

Comment:
useRef instead?

Reply:
Phil and I discussed, and decided it is not worth re-writing the component in React Hooks at this time, though this would definitely be a better coding solution.